incubator/java-openliberty/templates/kafka/README.md

```diff
@@ -0,0 +1,78 @@
+# Kafka template for Open Liberty
+
+This template can be used to develop Liberty applications that connect to Kafka by using MicroProfile Reactive messaging. A simple `StarterApplication` is included that enables basic production and consumption of events. 
+
+
+## Getting Started with the StarterApplication.
+
+### 1. Create a new folder and initialize it using appsody init:
+
+
+```
+mkdir test-appsody-reactive
+cd test-appsody-reactive
+appsody init java-openliberty reactive
+```
+
+### 2. Start Kafka and Zookeeper
+
+In order to run the `StarterApplication` you must start Kafka and Zookeeper containers. Use the `docker-compose.yaml` that is provided in the template.
+
+
+Start docker compose with the following command:
+
+```docker-compose -f docker-compose.yaml up```
+
+If you run `docker network list`, you should see a new network with the name of your project directory and the word `_default` appended. For example, `test-appsody-reactive_default`.
+
+Alternatively, if you want to connect to a Kafka broker elsewhere, edit `src/main/resources/META-INF/microprofile-config.properties` and set the value of the `mp.messaging.connector.liberty-kafka.bootstrap.servers` property to the host and port number of the your broker.
+
+### 3. Run the Appsody application in the new network
+
+Your Appsody application must be run in the same network as Kafka.
+
+Run the application using the following command:
+
+```appsody run --network test-appsody-reactive_default```
```

Comment:
As above, use `--docker-options` to show people how to set `MP_MESSAGING_CONNECTOR_LIBERTY_KAFKA_BOOTSTRAP_SERVERS`